process/incubation.md

```diff
@@ -39,8 +41,27 @@ For a pitch to be moved into the Proposal stage, it must be endorsed by at least
 * Social media accounts (optional)
 * Community size and any existing sponsorship
 
+Once a propsal PR is submitted, the SSWG will assign a review manager during it's bi-weekely meeting. The review manager responsibilities include:
+
+* Review the PR 
+  * Vaidate structure and language.
+  * Make sure the implementation is in place.
+* Update the PR
+  * Assign a number.
+  * Assign a review manager.
+  * Set status to "Active Review + Date range for review"
+* Secure approval from the PR author to merge changes above and merge them.
+* Publisha a form post in the [server proposals area](https://forums.swift.org/c/server/proposals) soliciting feedback from the community.
+* Watch the forum thread for conduct or off-topic issues, and make sure author is engaged.
+* Once the review period is over, summarize the key takeaways to the SSWG in writing.
+
 The SSWG votes on pending proposals on a bi-weekly cadence, with the goal of voting on at least two proposals per month.
 
+After the vote, the review manager will:
+* Annoucnce the vote results in the review thread.
+* Update the propsal's status based on the vote result.
+* Close ithe review thread.
```

Comment:
```suggestion
1. Announce the vote results in the review thread.
1. Update the Proposal's status based on the vote result.
1. Close the review thread.
```

Reply:
languages changes seem fine but we lost the header in L60 and not sure about `*` to `1.`

Reply:
Line 60 should still include `After the vote, the review manager will:`?

GitHub shows that my suggestion only applies to Lines 61-63, L60 will remain as-is in the tree.

Regarding the switch to numbered list, this is mostly a technical writing nitpick. As the header implies a discrete list of steps the review manager will take, they should be numbered in the order. Above where we outline responsibilities, they're not in any particular order aside from what you (the author) decided made the most hierarchical sense.